### PR TITLE
Adds that one meme anvil to the backrooms but removes the effects

### PR DESCRIPTION
**Pull request by cowbot92**
All credit goes to the original PR https://github.com/yogstation13/Yogstation/pull/21798

# Document the changes in your pull request

Adds the april fools anvil to the backrooms as a joke

# Why is this good for the game?

it'll be funny

# Testing

https://github.com/yogstation13/Yogstation/pull/21798

# Spriting

https://github.com/yogstation13/Yogstation/pull/21798

# Wiki Documentation

https://github.com/yogstation13/Yogstation/pull/21798

# Changelog

:cl:
rscadd: Adds anvil to the backrooms (not the wiki guy)
soundadd: Adds anvil sounds
imageadd: Adds anvil icons
mapping: maps some anvil
experimental: This is experimental
/:cl:

### CODE REVIEW

**Comment by cowbot92** [comment]
actually this broke my vs code so we'll see if I can even fix that

**Comment by cowbot92** [comment]
chat we're so back

**Comment by MommaDragon** [comment]
So, this just makes the anvil appear in the backrooms, right? Does it just do cosmetic changes (name and color), or will it still randomly give enchants with functions, including summoning things? Hopefully only the former.

File: code/game/objects/items.dm [hunk 1]
```diff
@@ -1,6 +1,6 @@
 GLOBAL_DATUM_INIT(fire_overlay, /mutable_appearance, mutable_appearance('icons/effects/fire.dmi', FIRE))
 
-GLOBAL_VAR_INIT(rpg_loot_items, FALSE)
+GLOBAL_VAR_INIT(rpg_loot_items, TRUE)
 // if true, everyone item when created will have its name changed to be
 // more... RPG-like.
```
Comment (Moltijoe) [inline code/game/objects/items.dm:5]: hm?

**Comment by cowbot92** [comment]
I need time to do this after the hurricane

**Pull request closed without merging**